src/main/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ set(GenomicsDB_library_sources
     cpp/src/utils/lut.cc
     cpp/src/utils/known_field_info.cc
     cpp/src/utils/vid_mapper.cc
+    cpp/src/utils/vid_mapper_sql.cc
```

Comment:
This is not necessary. There is a macro enveloping the code which prevents code from compiling
if libdbi is not installed. Should we remove that check and place this check instead?